distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -1,3 +1,230 @@
+
+On the *Result Preview* pane, view a summary of information about the selected resource.
+The *Details* tab provides a list of specific metadata.
+
+<<_viewing_revision_history,History>>:: view revision history of this resource.
+<<_editing_associations_on_a_record,Associations>>:: view or edit the relationship(s) between this resource and others in the catalog.
+<<_viewing_metadata_quality,Quality>>:: View the completeness and accuracy of the metadata for this resource.
+<<_exporting_a_record,Actions>>:: Export the metadata/resource to a specific format.
+<<_archiving_a_record,Archive>>:: Remove the selected record from standard search results.
+<<_overwriting_a_record,Overwrite>>:: Overwrite a record.
+
+==== Editing Records
+
+Records can be edited from the *Summary* or *Details* tabs.
+
+==== Editing Associations on a Record
+
+Update relationships between records through *Associations*.
+
+. Select the desired record from the *Results* tab.
+. Select the *Associations* tab.
+. Select *Edit*.
+. For a new association, select *Add Assorciation*.
+.. Select the related record from either the *Parent* or *Child* menu.
+.. Select the type of relationship from the *Relationship* menu.
+.. Select *Save*.
+. To edit an existing association, update the selections from the appropriate menu and select *Save*.
+
+==== Viewing Revision History
+
+View the complete revision history of a record.
+
+. Select the desired record from the *Results* tab.
+. Select the *History* tab.
+.. Select a previous version from the list to preview.
+.. If desired, select *Revert to Selected Version* to undo changes made after that revision.
+
+==== Viewing Metadata Quality
+
+View and fix issues with metadata quality in a record.
+
+[NOTE]
+====
+Correcting metadata issues may require heightened permissions.
+====
+
+. Select the desired record from the *Results* tab.
+. Select *More*.
+. Select *Quality*.
+. A report is displayed showing any issues:
+.. Metacard Validation Issues.
+.. Attribute Validation Issues.
+
+==== Exporting a Record
+
+Export and view a resource and/or its metadata into another format.
+
+. Select the desired record from the *Results* tab.
+. Select *More*.
+. Select *Actions*.
+. Available export formats are listed. Select desired format.
+. Export opens in a new browser tab. Save, if desired.
+
+==== Archiving a Record
+
+To remove a record from active search results, archive it.
+
+. Select the desired record from the *Results* tab.
+. Select *More*.
+. Select *Archive*.
+. Confirm archiving the record.
+
+==== Restoring an Archived Record
+
+Restore an archived record to return it to active search results.
+
+. Select the *Search Archived* option from the query options menu.
+. Select the desired record from the *Results* tab.
+. Select *Archive*.
+. Confirm restoring the record.
+
+==== Overwriting a Record
+
+Replace a resource.
+
+. Select the desired record from the *Results* tab.
+. Select *More*.
+. Select *Overwrite*.
+
+
```

Comment:
A `Settings` section that is missing is the gear menu, that holds the `Theme`, `Notifications`, `Map`, `Visualization`, and `Hidden`. 

![screenshot - settings](https://cloud.githubusercontent.com/assets/16792154/21824057/f3d6ceb2-d73a-11e6-903e-9443a32b89cb.png)

Be sure to include how to Unhide Blacklisted items:
![screen shot 2017-01-10 at 1 43 17 pm](https://cloud.githubusercontent.com/assets/16792154/21824068/0605b29c-d73b-11e6-8a54-a1faa536b388.png)